app/models/user.rb

```diff
@@ -714,6 +714,10 @@ def can_edit_user?(user)
     self == user || can_view_all_users? || organizer_for?(user)
   end
 
+  def can_edit_any_user?
+    admin? || any_kind_of_delegate? || results_team? || communication_team?
```

Comment:
@danieljames-dj should `communication_team` have edit permissions in your view? In my mind, they should only need view permissions.

Reply:
@dunkOnIT WCT has been regularly handling wca id re assigning tasks which are trivial and can be performed by WRT too. We in fact had a discussion with what should be done but no conclusive solutions. @danieljames-dj should there be another WRT discussion on this? Perhaps if both teams vote together on what should be done, then we maybe will get a permanent solution

Reply:
@dunkOnIT Yes, currently, they handle requests like "change WCA ID to new account" themselves instead of forwarding them to WRT.

Reply:
@lnzainn We can think of having another discussion for sure. But I guess that won't block this PR to be submitted anyway because this PR doesn't change any permissions related stuffs. :-)

Reply:
Ah ok I see - and is this preferred by WRT, or is it worth re-evaluating if this is the best arrangement? (I assume it's preferred by WRT, but sometimes things can just be the way they are because no one's thought about them in a while - eg, WRC having permission to view registrations :P

Reply:
Not speaking of behalf of WRT, but when someone from wct made a rookie mistake of forwarding such request to wrt, then response from wrt was 'this can be done by wct themselves, wrt intervention is not required' 

Btw I'm that rookie who made this mistake when I had newly joined the team 😂